src/usr/local/www/diag_confbak.php

```diff
@@ -206,9 +206,18 @@
 print($form);
 
 if (is_array($confvers)) {
-	print_info_box(gettext('To view the differences between an older configuration and a newer configuration, ' .
-						   'select the older configuration using the left column of radio options and select the newer configuration in the right column, ' .
-						   'then press the "Diff" button.'));
+?>
+<div>
+	<div id="infoblock">
```

Comment:
If I remove the infoblock stuff then sure, it becomes just and ordinary info box.
But then if I "x" it, it goes away completely and I can't open it again (of course if I refresh the whole page it comes back).
1) Do we want info boxes like that? That can be dismissed and they are gone.
2) Or do we always want ones that have the little "i" icon and can be toggled on and off?
If (2), then (maybe) sometimes we would like them to be open on initial page load.